Dockerfile

```diff
@@ -0,0 +1,33 @@
+# For more information, please refer to https://aka.ms/vscode-docker-python
+FROM python:3.8-slim-buster
+
+# Keeps Python from generating .pyc files in the container
+ENV PYTHONDONTWRITEBYTECODE=1
+
+# Turns off buffering for easier container logging
+ENV PYTHONUNBUFFERED=1
+
+RUN apt update && apt-get -y install sudo
+RUN apt-get install -y wget
+RUN wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc
+RUN apt-get install -y postgresql
+
```

Comment:
@annahramova97 is it possible to also set up the database in the docker file and when the users will set up the database for programming everything will already be set up?

Reply:
We can set it up using docker-compose, which will build both cocoa and postgresql as 2 services which are able to communicate